src/check_release.rs

```diff
@@ -419,7 +421,7 @@ pub(super) fn run_check_release(
             .shell_print(
                 "Checked",
                 format_args!(
-                    "[{:>8.3}s] {} checks; {} passed, {} skipped",
+                    "[{:>8.3}s] {} checks; {} pass, {} skip",
```

Comment:
Another idea for improving the "feel" of not using past tense: replace the semicolon with a colon, so it implies that we're expanding upon the `{} checks` bit.
```suggestion
                    "[{:>8.3}s] {} checks: {} pass, {} skip",
```

This should produce something like `[ 123.456s] 80 checks: 75 pass, 3 error, 2 warn, 1 skip`